rs/ledger_suite/icp/ledger/Cargo.toml

```diff
@@ -21,12 +21,14 @@ dfn_http_metrics = { path = "../../../rust_canisters/dfn_http_metrics" }
 dfn_protobuf = { path = "../../../rust_canisters/dfn_protobuf" }
 ic-base-types = { path = "../../../types/base_types" }
 ic-canister-log = { path = "../../../rust_canisters/canister_log" }
+ic-cdk = { workspace = true }
```

Comment:
I remember that Roman was very much against using ic-cdk and dfn_core at the same time in the same canister. Are we sure we want to proceed with this?